Content.Server/AME/Components/AMEControllerComponent.cs

```diff
@@ -285,6 +299,11 @@ private void InjectSound(bool overloading)
         {
             SoundSystem.Play(_injectSound.GetSound(), Filter.Pvs(Owner), Owner, AudioParams.Default.WithVolume(overloading ? 10f : 0f));
         }
+
+        private void NoFuelSound()
+        {
+            SoundSystem.Play(_noFuelSound.GetSound(), Filter.Pvs(Owner), Owner, AudioParams.Default.WithVolume(-2f));
```

Comment:
```suggestion
            _entities.EntitySysManager.GetEntitySystem<SharedAudioSystem>().PlayPvs(_noFuelSound, Owner, AudioParams.Default.WithVolume(-2f));
```
this is oldcode and not ECS, hence the super weird call here. Ideally this needs to be refactored, but it's better to not use obsolete methods in any case.